### DISCUSSION TITLE
Share the results card with information about your play as image?

### DISCUSSION THREAD

**Discussion by puyoxyz** [ppy/osu#17102] [Ideas]
Sorry if this has already been suggested, but, these cards on the results screen look very cool, so, not sure how hard this would be to implement, but it would be very epic if there was a button to share this card as an image, with a transparent background (instead of the blurred map bg behind it), and either copy it to your clipboard or put it in the osu/screenshots folder ? Especially since you can't share osu!web links for lazer scores (no "see details" button on them in recent plays section on user profile)

![image](https://user-images.githubusercontent.com/15311104/156831791-f1c53691-2a4e-4355-b5a6-55cf06efbd63.png)

**Comment by frenzibyte** [comment]
Would require a good amount of effort to the point it may just be considered unfeasible. Adding the ability to link osu!lazer scores on the other hand is to be added eventually, and would probably be enough for this.
